### ISSUE TITLE
gdl-1.0.1 segfaulting on "reset"

### ISSUE THREAD

**Issue by dirteat** [gnudatalanguage/gdl#1307]
Hi,
trying to do a reset in the console produces a segfault. Tested on gdl-1.0.1 and gdl-1.0.0, only gdl-1.0.1 has the issue.

Let me know if you need  a complete bt.

```
.reset
[hostname:2888624] *** Process received signal ***
[hostname:2888624] Signal: Segmentation fault (11)
[hostname:2888624] Signal code: Address not mapped (1)
[hostname:2888624] Failing at address: (nil)
[hostname:2888624] [ 0] /lib64/libpthread.so.0(+0x13180)[0x7fb6d27fc180]
[hostname:2888624] *** End of error message ***
Segmentation fault (core dumped)
```

Running the same within gdb gives:

```
Thread 1 "gdl" received signal SIGSEGV, Segmentation fault.
0x0000000000bc809b in GDLWidget::UnInit() ()
```

Cheers,
Chris.

**Comment by GillesDuvert** [comment]
good find, this bug.

**Comment by GillesDuvert** [comment]
thanks, solved in #1310
